learning/tour-of-beam/terraform-v2/environments/uat/variables.tf

```diff
@@ -0,0 +1,12 @@
+variable "bucket_name" {
+  description = "Bucket name to store function source code"
+}
+variable "project_id" {
+  description = "Our GCP Project"
+}
+variable "service_account_id" {
+  description = "Name of SA to run Cloud Function"
+}
+variable "region" {
+  default = "europe-west1"
```

Comment:
Bad default choice, it is better be the same as in Playground: `us-central1`

Reply:
Action: To rename default "region"